### DISCUSSION TITLE
Can't install the latest pnpm version (8.3.1) using homebrew

### DISCUSSION THREAD

**Discussion by ubmit** [pnpm/pnpm#6475] [Q&A]
I ran into issues using `pnpm@8.2.0` along with `node@20.0.0` and from what I could find, a fix was released in version `8.3.1`. Considering that, I tried to run `brew upgrade pnpm` but according to brew I was already at the latest version.

When I ran `brew info pnpm` I realized that the `8.3.1` is available:
```sh
==> pnpm: stable 8.3.1 (bottled)
📦🚀 Fast, disk space efficient package manager
https://pnpm.io/
Conflicts with:
  corepack (because both installs `pnpm` and `pnpx` binaries)
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/pnpm.rb
License: MIT
==> Dependencies
==> Caveats
pnpm requires a Node installation to function. You can install one with:
  brew install node
==> Analytics
install: 643 (30 days), 29,707 (90 days), 214,978 (365 days)
install-on-request: 644 (30 days), 29,720 (90 days), 214,982 (365 days)
build-error: 0 (30 days)
```

However, I can't install it. I've already tried running `brew install pnpm@8.3.1` and it looks like brew doesn't find the version. Here is the output for `brew search pnpm`:
```sh
==> Formulae
pnpm
```

How can I install the latest version using homebrew?

**Comment by piersolenski** [comment]
Did you find a way to solve this @ubmit?
